types/fullcalendar/index.d.ts

```diff
@@ -995,7 +995,7 @@ declare global {
         /**
          * Immediately switches to a different view.
          */
-        fullCalendar(method: 'changeView', viewName: string): void;
+        fullCalendar(method: 'changeView', viewName: string, dateOrRange: moment.Moment | Date | string | TimeRange): void;
```

Comment:
[dateOrRange is optionnal](https://fullcalendar.io/docs/views/changeView/). Please change `dateOrRange` to `dateOrRange?`

Reply:
Thanks for that @hmil , it has been changed to optional.